stanford_bean_types.features.field_base.inc

```diff
@@ -912,21 +1027,6 @@ function stanford_bean_types_field_default_field_bases() {
     'type' => 'list_text',
   );
 
-  // Exported field_base: 'field_s_smc_icon'.
-  $field_bases['field_s_smc_icon'] = array(
```

Comment:
This shouldn't be removed. you probably have something missing for this to remain.